guides/common/modules/con_ipv4-and-ipv6-requirements.adoc

```diff
@@ -0,0 +1,45 @@
+[id="ipv4-and-ipv6-requirements_{context}"]
+= IPv4 and IPv6 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+
+ifdef::satellite[]
+[NOTE]
+====
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+====
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* An IPv6 loopback must be configured on the base system.
+The loopback is typically configured by default.
+Do not disable it.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+ifdef::satellite[]
++
+For a supported way to disable the IPv6 protocol, see link:https://access.redhat.com/solutions/5045841[How do I disable the IPv6 protocol on Red Hat Satellite and/or Red Hat Capsule server?] in Red{nbsp}Hat Knowledgebase.
+endif::[]
+
+The following requirements apply to installations in an IPv6 network:
+
+* If you intend to use {Project} to provision hosts, you must deploy an external DHCPv6 server and cofigure it manually to communicate with the network boot process and to manage IP address assignment because {Project} cannot integrate with a DHCPv6 server and manage its configuration.
+For more information about DHCPv6 server configuration, see {ProvisioningDocURL}options-in-unmanaged-dhcpv6[Options in unmanaged DHCPv6] in _{ProvisioningDocTitle}_.
+ifdef::satellite[]
+* You must deploy an external HTTP proxy server that supports both IPv4 and IPv6.
+This is required because Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this proxy to pull content into the {Project} on your IPv6 network.
+You must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy server as the default proxy.
+endif::[]
+ifndef::satellite[]
+* Optional: If you rely on content from IPv4-only networks and are deployed on IPv6-only, you can use an external dual stack HTTP proxy server.
+In this setup, you must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy server as the default proxy.
+endif::[]
++
+For more information, see {InstallingServerDocURL}adding-a-default-http-proxy_{project-context}[Adding a Default HTTP Proxy to {Project}].
+
+Although {Project} provisioning templates include IPv6 support for PXE and HTTP (iPXE) provisioning, the only tested and certified provisioning workflow is the UEFI HTTP Boot provisioning.
+This limitation only relates to users who plan to use {Project} to provision hosts.
```

Comment:
```suggestion
```
I think that these lines no longer apply because of #3532 (TFTP ~ PXE). I think we can just remove them.

@ShimShtein Can you please confirm? Or is there another limitation?

Reply:
I can confirm that I am working on enabling PXE provisioning in IPv6 networking.

As far as limitations go - I don't see anything yet